source/forth3/.github/workflows/ci.yml

```diff
@@ -0,0 +1,165 @@
+name: CI
```

Comment:
This should eventually be integrated into mnemos' CI. It's pretty important IMO to preserve regular + miri testing.